packages/ffe-account-selector-react/src/index.d.ts

```diff
@@ -8,10 +8,10 @@ export interface Account {
 }
 
 interface ListElementBodyProps {
-    item: Account;
-    isHighlighted: boolean;
-    locale: string;
-    dropdownAttributes: string[];
+    item?: Account;
```

Comment:
vorfor blir denne optional? Man får vell ikke laget en list element body utan en Account?

Reply:
`ListElementBodyProps` er parameteren til den user-defined callbacken `AccountSelector.listElementBody()`. For å gi utviklerne frihet er alle props på `ListElementBodyProps` valgfri, selv om det ikke gir mening å ikke bruke `item`.

Reply:
oki